nominees/opentelerehab.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "OpenTeleRehab",
+  "aliases": [
+    ""
+  ],
+  "description": "Open Source, Multidisciplinary Telerehabilitation Software",
+  "website": "https://www.opentelerehab.com",
+  "license": [
+    {
+      "spdx": "Apache-2.0",
+      "licenseURL": "https://github.com/OpenTeleRehab/therapist-web-app/blob/main/LICENSE"
+    },
+    {
+      "spdx": "Apache-2.0",
```

Comment:
@nathanbaleeta - I don't think the license should be listed 3 times. We should use the license from the GitHub repo listed below. They could have multiple entries for multiple repositories if that is appropriate.

Reply:
Agreed

Reply:
Fixed in commit 169e04